spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -208,24 +208,145 @@ $ tail -f /var/folders/wn/8jxm_tbd1vj28c8vj37n900m0000gn/T/spring-cloud-dataflow
+----
+
+Below is the white listed properties for the app `log`:
+
+[source,bash]
+----
+╔══════════════════════════════╤══════════════════════════════╤══════════════════════════════╤══════════════════════════════╗
+║         Option Name          │         Description          │           Default            │             Type             ║
+╠══════════════════════════════╪══════════════════════════════╪══════════════════════════════╪══════════════════════════════╣
+║log.name                      │The name of the logger to use.│<none>                        │java.lang.String              ║
+║log.level                     │The level at which to log     │<none>                        │org.springframework.integratio║
+║                              │messages.                     │                              │n.handler.LoggingHandler$Level║
+║log.expression                │A SpEL expression (against the│payload                       │java.lang.String              ║
+║                              │incoming message) to evaluate │                              │                              ║
+║                              │as the logged message.        │                              │                              ║
+╚══════════════════════════════╧══════════════════════════════╧══════════════════════════════╧══════════════════════════════╝
+----
+
+The application properties for `time` and `log` apps can be specified at the time of `stream` creation as follows:
+
+[source,bash]
+----
+dataflow:> stream create --definition "time --fixed-delay=5 | log --level=WARN" --name ticktock
+----
+
+Note that the properties `fixed-delay` and `level` defined above for the apps `time` and `log` are the `short-form` property names provided by the shell completion.
+These `short-form` property names are applicable only for the white-listed properties and in all other cases, only `fully qualified` property names should be used.
+
+
+==== Passing application properties when deploying stream
+
+The application properties can also be specified when deploying the stream. When specified during deployment, these application properties can either be specified as
+ `short-hand` property names (applicable for white-listed properties) or fully qualified property names. The application properties should have the prefix "app.<app name/label>".
+
+For example, the stream
+
+[source,bash]
+----
+dataflow:> stream create --definition "time | log" --name ticktock
+----
+
+can be deployed with application properties using the`short-hand` property names:
+
+[source,bash]
+----
+dataflow:>stream deploy ticktock --properties "app.time.fixed-delay=5,app.log.level=ERROR"
+----
+
+When using the app label,
+
+[source,bash]
+----
+stream create ticktock --definition "a: time | b: log"
+----
+
+the application properties can be defined as:
+
+[source,bash]
+----
+stream deploy ticktock --properties "app.a.fixed-delay=4,app.b.log.level=ERROR"
```

Comment:
here you have `app.b.log.level` whereas the example above was `app.log.level`... if the goal is to be consistent across both then shouldn't this one be `app.b.level`? (just like the previous property is simply `app.a.fixed-delay`)